### DISCUSSION TITLE
lab6 simulation problem

### DISCUSSION THREAD

**Discussion by weixiang1118** [bol-edu/HLS-SOC-Discussions#152] [General]
![1](https://github.com/bol-edu/HLS-SOC-Discussions/assets/81802711/b88d72f7-b0ff-4e61-8ad7-8d06e39b05d0)
如圖 再跑個別模擬的時候，都會發生這個問題，請問是為甚麼?
謝謝

**Comment by LingChiYang** [comment]
同學好

缺少folder
https://github.com/bol-edu/caravel-soc_fpga-lab/tree/main/lab-wlos_baseline/cvc-pdk
已經更新repo，可以請同學在pull下來或是從別的lab複製過來
